css/index.css

```diff
@@ -1 +1,468 @@
-/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
\ No newline at end of file
+/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', sans-serif;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+/* Body */
+
+.wrapper {
+    max-width: 880px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+p, h2 {
+    font-size: 1.6rem;
+}
+
+nav, header, section {
+    display: flex;
+    padding: 34px 0px 0px;
+}
+
+/* Disabled links */
+
+nav a, header a, section a {
+    text-decoration: none;
+}
+
+header a, section a {
+    color: black;
+}
+
+
+/*Navigation */
+
+nav {
+    justify-content: space-between;
+    
+}
+.nav {
+    display: flex;
+    flex-wrap: wrap;
+}
+nav a {
+    color: #bebebe;
+    padding: 0px 30px;
```

Comment:
This padding (and related padding in your media queries) causes an issue between your mobile and tablet views. However, without it, your desktop view doesn't look good. You don't need the padding, but to keep the desktop looking nice, make the `.nav.menu` and `.nav.logo` containers have percentage-based widths, set them to `display: flex` and then position the elements inside of them accordingly.